.github/workflows/build.yml

```diff
@@ -9,15 +9,18 @@ on:
 
 jobs: 
   build:
-    name: Build
+    name: Build ${{ matrix.configuration }}
     runs-on: windows-2022
+    strategy:
+      matrix:
+        configuration: ['Debug', 'Release']
```

Comment:
I haven't seen any `#if DEBUG` in this package, so just building Release configuration should be fine